src/pydicom/dataelem.py

```diff
@@ -31,8 +34,10 @@
 from pydicom.tag import Tag, BaseTag
 from pydicom.uid import UID
 from pydicom import jsonrep
+from pydicom.util.buffers import buffer_assertions
 import pydicom.valuerep  # don't import DS directly as can be changed by config
 from pydicom.valuerep import (
+    BUFFERED_VRS,
     PersonName,
```

Comment:
A real nitpick, but perhaps this should be `BUFFERABLE_VRS`?  They are not actually buffered, necessarily.

Reply:
Good point - changed.